portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Class representing a comment.
+ */
+package com.google.sps.data;
+import java.util.Date;
+
+
+public class Comment {
+    private String name; // name of the commenter
+    private String content; // content of the comment
+    private final Date date = new Date(); // date the comment was left
```

Comment:
I'm not as familiar with the subtleties of java, but if this were python, setting the date outside the constructor would make the date be the same for all classes. I think it would maybe be the time the class gets compiled. I'd look into how java handles things and maybe move this into the constructor.